README.md

```diff
@@ -75,3 +75,13 @@ Topic:topic2    PartitionCount:5        ReplicationFactor:1     Configs:retentio
 ```
 
 Any changes to the `test-topics.yml` file will be applied to the existing topics at each subsequent run.
+
+## Docker
+
+The Docker image is available from Docker Hub.
```

Comment:
Does this information (or some subset of it) need to be put into or directly below the existing `Download` section?